### PR TITLE
Suggested some hints for the Repl

### PR DESCRIPTION
**Pull request by jrd656**
Add hints to the bottom of the guide, and change the task order to emphasise to read through the guide first.

### CODE REVIEW

**Comment by Karyum** [comment]
@jrd656 should we close the previous pull request then? and want to keep updating this one ?

File: coursebook/week-2/dom-manipulation-ws.md [hunk 1]
```diff
@@ -132,3 +134,65 @@ We can access _properties_ of DOM elements using javscript.
...
+console.log(1);
+
+let introductionElement = document.querySelector('#introduction');
+
+if (introductionElement.classList.contains('highlight')) {
+    introductionElement.classList.remove('highlight')}
+  }
+  else {
+    introductionElement.classList.add('highlight');
+    }
```
Comment (Karyum) [inline coursebook/week-2/dom-manipulation-ws.md:193]: using the `toggle` method would be ideal fora situation like this.

```javascript
introductionElement.classList.toggle('highlight')
```

[the toggle method](https://alligator.io/js/classlist/#toggle)

File: coursebook/week-2/dom-manipulation-ws.md [hunk 2]
```diff
@@ -132,3 +134,65 @@ We can access _properties_ of DOM elements using javscript.
...
+
+<details><summary>Solution</summary>
+<p>
+
+```javascript
+
+var toggleHighlight = function () {
+
+// Code here
+console.log(1);
```
Comment (Karyum) [inline coursebook/week-2/dom-manipulation-ws.md:184]: i think we can delete these 2 lines from here

**Review by Karyum — CHANGES_REQUESTED** [review]

**Comment by jrd656** [comment]
Yeah I've tried both toggle and add and remove. I was going for add and remove as those were the words used in the question (so I thought it might be steering us to use those methods).

image

I've just tried another method that worked on Moris's computer and I still get the same error.

**Comment by Karyum** [comment]
hmm ok, it's a different error from the one that was sent in slack, the error also is coming from the tests file (check the second line of the error message) rather than the exercise file.

i've removed it. [click me!](https://repl.it/@Karyum1/DOM-manipulation-Challenge-A)

**Comment by jrd656** [comment]
Haha! Wooooooo!!! FINALLY COMPLETED QUESTION 1!!!!!

Cheers mate

**Comment by Karyum** [comment]
changes moved to #9

**Pull request closed without merging**